plugin/src/generators/sentry/utils/generate-sentry-expo.ts

```diff
@@ -0,0 +1,155 @@
+import { addDependenciesToPackageJson, Tree } from '@nx/devkit';
+import { tsquery } from '@phenomnomnominal/tsquery';
+import {
+  createPrinter,
+  factory,
+  ObjectLiteralExpression,
+  SyntaxKind,
+  addSyntheticLeadingComment,
+  ArrayLiteralExpression,
+} from 'typescript';
+import { createObjectLiteralExpression } from './create-object-literal-expression';
+import { SentryGeneratorSchema } from '../schema';
+import { updateFile } from 'plugin/src/shared/utils';
+
+const expoAppDependencies = {
+  '@sentry/react-native': '~6.1.0',
+};
```

Comment:
please move it to our dependencies [file](https://github.com/RonasIT/nx-generators/blob/main/plugin/src/shared/dependencies.ts)